### DISCUSSION TITLE
Keep existing task defs active when deploying update of a fargate-loadbalanced-ecs-service

### DISCUSSION THREAD

**Discussion by diasdev** [aws/aws-cdk#25225] [Q&A]
When using CDK stack deployment to update an ECS service (ApplicationLoadBalancedFargateService construct), the deployment will create a new task definition (which is expected) and then inactivate pior task defs, which makes rolling back to a previous task def revision impractical.
Is there any argument I can pass to ApplicationLoadBalancedFargateService that prevents cdk/cfn to inactivate older task definitions when creating a new revision to update the ecs service?

**Comment by michaelcummings12** [comment]
Just specify the job definition name (without the revision) and your task will always use the latest!

**Comment by 39otrebla** [comment]
@diasdev did you find a solution? I'm stuck on the same issue, but can't find any option in the latest CDK version (Typescript).
